Missing `jszip` dependency in `phoenix-ui-components`

When trying to use `yarn` instead of `npm` to create an event display for an event display, I ran into the following issue (see also https://github.com/HSF/phoenix/pull/668#pullrequestreview-2185039443 for more details).

```bash
ng new event-display-app --style scss --routing true --interactive false
cd event-display-app
yarn set version berry # as recommended by main phoenix documentation
yarn add phoenix-ui-components
yarn add phoenix-event-display
ng generate component main-display
```
Then doing the necessary adaptions as described and finally
```bash
yarn start
```
leads to
```console
[...]
✘ [ERROR] Could not resolve "jszip"

    .yarn/__virtual__/phoenix-ui-components-virtual-7124f6bf67/5/.yarn/berry/cache/phoenix-ui-components-npm-2.16.0-baa536ee41-10c0.zip/node_modules/phoenix-ui-components/dist/fesm2022/phoenix-ui-components.mjs:40:18:
      40 │ import JSZip from 'jszip';
         ╵                   ~~~~~~~

  The Yarn Plug'n'Play manifest forbids importing "jszip" here because it's not listed as a
  dependency of this package:

    .pnp.cjs:10935:31:
      10935 │         "packageDependencies": [\
            ╵                                ~~

  You can mark the path "jszip" as external to exclude it from the bundle, which will remove this
  error and leave the unresolved path in the bundle.
```

This can be fixed with
```yaml
nodeLinker: node-modules
```
in `.yarnrc.yml` but maybe there is a way to also make this work with yarn pnp?

--- a/packages/phoenix-ng/package.json
+++ b/packages/phoenix-ng/package.json
@@ -36,6 +36,7 @@
     "@angular/router": "^17.3.11",
     "css-element-queries": "^1.2.3",
     "cypress-plugin-snapshots": "https://github.com/meinaart/cypress-plugin-snapshots.git",
+    "jszip": "^3.10.1",
     "phoenix-event-display": "^2.16.0",
     "phoenix-ui-components": "^2.16.0",
     "qrcode": "1.5.3",

